pfp/index.html

```diff
@@ -687,11 +895,277 @@ <h2>Packing Results:</h2>
 
 
 
-    <!-- Custom scripts -->
+
+
+    <!-- Luxon javascript library for working with dates/times (using for pdf filename generation) -->
```

Comment:
Day.js would've probably been a much simpler solution for what I wanted, and required less code. Might look into switching in the future if there's ever a reason, or maybe I'll start with it in any new projects.